### ISSUE TITLE
Regex sanitization might be flawed

### ISSUE THREAD

**Issue by Moon-0xff** [Moon-0xff/gnome-mpris-label#61]

**Comment by abisammy** [comment]
Was thinking about this...

Might be better to implement filters in a better way, for example a more "todo list" style such as:
https://www.w3schools.com/howto/howto_js_todolist.asp
![image](https://github.com/Moon-0xff/gnome-mpris-label/assets/75337261/67ecdfac-60fd-415e-957f-163cf780a8ae)

I'm pretty sure in gnome js, settings can be stored as arrays, so the blacklist can be stored as an array, additionaly the todo list would make it more user friendly.

This removes the annoyances of having a regex, and will fix this bug.

**Comment by Moon-0xff** [comment]
Yes, it would remove the problems of dealing with the values, but implementing the layout itself might be a problem on it's own.

We could try to implement something like this but at this point in time it's better to wait until we can ditch GTK3 compatibility.
